cmd/create.go

```diff
@@ -24,25 +28,29 @@ func init() {
 	createCmd.Flags().StringP("name", "n", "", "optionally specify an explicit name for the serive, overriding path-derivation. $FAAS_NAME")
 	createCmd.Flags().StringP("registry", "r", "quay.io", "image registry (ex: quay.io). $FAAS_REGISTRY")
 	createCmd.Flags().StringP("namespace", "s", "", "namespace at image registry (usually username or org name). $FAAS_NAMESPACE")
+	createCmd.Flags().StringP("context", "x", embedded.DefaultContext, "Function context (ex: 'http','events'). $FAAS_CONTEXT")
+	createCmd.Flags().StringP("templates", "", filepath.Join(configPath(), "faas", "templates"), "Extensible templates path. $FAAS_TEMPLATES")
 	createCmd.RegisterFlagCompletionFunc("registry", CompleteRegistryList)
 }
 
 // The create command invokes the Service Funciton Client to create a new,
 // functional, deployed service function with a noop implementation.  It
 // can be optionally created only locally (no deploy) using --local.
 var createCmd = &cobra.Command{
-	Use:        "create <language>",
-	Short:      "Create a Service Function",
-	SuggestFor: []string{"init", "new"},
+	Use:               "create <language>",
+	Short:             "Create a Service Function",
```

Comment:
Nit: could this just be "Create a Function"?

Reply:
Thanks @lance ; If that seems more obvious to you, I can go with just "Functions" for now.  I will summarize the method to my madness in having called them thus and pop the question on Slack to see if there are others with an opinion.
Since your gut reaction was immediately that the term is redundant or unnecessary, I will go ahead and remove the prefix unless the team comes down squarely in favor of the full "Service Function".

Reply:
After getting some feedback from the team as well, Functions works for now, and we can kick the question about whether the 'faas' term itself is applicable, down the proverbial road.